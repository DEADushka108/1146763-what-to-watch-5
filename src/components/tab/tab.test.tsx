import React from 'react';
import renderer from 'react-test-renderer';
import Tab from './tab';

const mock = {
  children: <div/>,
};

it(`Should render Tab component correctly`, () => {
  const {children} = mock;
  const tree = renderer
    .create(
        <Tab>{children}</Tab>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
