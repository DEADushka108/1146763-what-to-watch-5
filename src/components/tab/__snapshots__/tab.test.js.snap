// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Tab component correctly 1`] = `<div />`;
